toolkit/tools/grapher/grapher.go

```diff
@@ -25,6 +26,20 @@ var (
 	strictGoals      = app.Flag("strict-goals", "Don't allow missing goal packages").Bool()
 	strictUnresolved = app.Flag("strict-unresolved", "Don't allow missing unresolved packages").Bool()
 
+	resolveCyles		 = app.Flag("resolve-cycles-from-repo", "Let grapher resolve cycles by marking rpms available in repo as remote").Bool()
+	outDir      = exe.OutputDirFlag(app, "Directory to download packages into.")
```

Comment:
>OutputDirFlag [](http://example.com/codeflow?start=19&length=13)

Repeated output dir flag - there should be only one. Is there an issue with the one we already have above?

Reply:
The first output flag is the actual output directory where grapher writes its output dot graph to. The repeater out-dir flag is added because it is mounted in the chroot when initialized by repocloner. This out-dir is not used by grapher but being passed as I'm reusing the function to initialize cloner. Hence the big list of args that are passed. I can rename the outDir to avoid confusion.